content/source/docs/enterprise-beta/saml/configuration.html.md

```diff
@@ -6,10 +6,23 @@ sidebar_current: "docs-enterprise2-saml-configuration"
 
 # Configuration
 
+SAML requires the configuration of both parties, the Identity Provider (IdP) and the Service Provider (SP) also sometimes referred to as Relying Party (RP). Terraform Enterprise is configured as the Service Provider.
```

Comment:
Relying -> Relaying

Reply:
I think its supposed to be "relying".
https://www.oasis-open.org/committees/download.php/21111/saml-glossary-2.0-os.html#Relying Party